api/src/main/java/datawave/webservice/dictionary/data/DefaultDataDictionary.java

```diff
@@ -37,13 +37,15 @@ public class DefaultDataDictionary extends DataDictionaryBase<DefaultDataDiction
     /*
      * Loads jQuery, DataTables, some CSS elements for DataTables, and executes `.dataTables()` on the HTML table in the payload.
      * 
-     * Pagination on the table is turned off, we do an ascending sort on the 2nd column (field name) and a cookie is saved in the browser that will leave the
+     * Pagination on the table is turned off, we do an ascending sort on the 1st column (field name) and a cookie is saved in the browser that will leave the
      * last sort in place upon revisit of the page.
      */
     private static final String DATA_TABLES_TEMPLATE = "<script type=''text/javascript'' src=''{0}jquery.min.js''></script>\n"
                     + "<script type=''text/javascript'' src=''{1}jquery.dataTables.min.js''></script>\n" + "<script type=''text/javascript''>\n"
-                    + "$(document).ready(function() '{' $(''#myTable'').dataTable('{'\"bPaginate\": false, \"aaSorting\": [[0, \"asc\"]], \"bStateSave\": true'}'); setTimeout( function() '{' $(''#myTable'').find(\"td\").css(\"word-break\", \"break-word\"); '}', 4000); '}');\n"
+                    + "$(document).ready(function() '{' $(''#myTable'').dataTable('{'\"bPaginate\": false, \"aaSorting\": [[0, \"asc\"]], \"bStateSave\": true'}'); $(''#myTable'').find(\"td\").css(\"word-break\", \"break-word\"); '}');\n"
```

Comment:
So the change here was to add this css immediately instead of after a 4 second delay?  Was there a reason for that?

Reply:
Yes I just removed the 4 second delay. The only reason I changed this was because it seemed unnecessary to wait 4 seconds as the table would be one format on loading the page then change after 4 seconds. I didn't see a need for this delay but I'm not sure maybe it was needed for some reason. If it is needed for some reason, I'm sure there is a better way to wait instead of waiting a constant amount of time.